test/runAll.spec.js

```diff
@@ -9,11 +10,18 @@ sgfMock.mockImplementation((params, callback) => {
 })
 
 const scripts = { mytask: 'echo "Running task"' }
+const globalConsoleTemp = global.console
 
 describe('runAll', () => {
+  beforeEach(() => {
+    global.console = makeConsoleMock()
+  })
   afterEach(() => {
```

Comment:
Let's remove this an instead switch `sgfMock.mockImplementation` to `sgfMock.mockImplementationOnce`.